src/azure-cli/azure/cli/command_modules/aro/_params.py

```diff
@@ -64,9 +64,11 @@ def load_arguments(self, _):
         c.argument('worker_vm_size',
```

Comment:
There is a diff in this file around `get_location_type(self.cli_ctx)` between here and ARO-RP.  Not sure which is right.  Please can you check?